pkg/bundle/read.go

```diff
@@ -40,7 +40,21 @@ func Open(ctx context.Context, name, baseDir, file string, opts *Options) (*Bund
 	)
 
 	if file == "" {
+		// check for both fleet.yaml and fleet.yml
 		file = filepath.Join(baseDir, "fleet.yaml")
+		if _, err := os.Stat(file); err == nil {
+			// fleet.yaml exists
+		} else if os.IsNotExist(err) {
+			// fleet.yaml does *not* exist
+			file = filepath.Join(baseDir, "fleet.yml")
+			if _, err := os.Stat("/path/to/whatever"); err == nil {
+				// fleet.yml exists
+			} else if os.IsNotExist(err) {
+				// fleet.yaml and fleet.yml does *not* exist
+				return nil, err
+			}
+		}
+
```

Comment:
Part of me almost wants to enforce the `.yaml` extension instead... Hm... Thoughts @StrongMonkey @kinarashah ?

Reply:
I don't feel too strongly either way, but it was something that just got me when i kept naming it `fleet.yml`.

Reply:
I would rather use fleet.yaml and not bother a different format like fleet.yml. It is just a convention that user will follow.

Reply:
My only argument is that generally `yml` and `yaml` are interchangeable. Fleet itself will register any `yml` resource files, so i would expect `fleet.yml` to be handled the same way.

Reply:
I support this update, as it can be really frustrating for new users that are used to using `.yml` extensions, to figure out why Fleet isn't behaving as expected, when the only "problem" is that they created `fleet.yml` instead of `fleet.yaml`.

Source: This was me for the first few days trying to learn how to use Fleet.